first_ruby/.browserslistrc

```diff
@@ -0,0 +1 @@
+defaults
```

Comment:
What does this mean?

Reply:
A tool to target specific browser version supported in frontend. The configuration controls the outputted JavaScript so that the code will be compatible with the configured browser. In this case, `defaults` are configured, the browsers which covers in this category can be listed using `npx browserlist defaults` command in the console.